scripts/npm/create-component.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+/* eslint-disable no-console */
+
+const { execSync } = require('child_process');
```

Comment:
`sync` 👎

Reply:
I get you, but what's the benefit of doing this asynchronously?

Reply:
in this particular case there probably no benefit, but it's not best practice, the power of node is in the event loop and async I/O

It's just me ranting you can safely ignore me 😄